### PR TITLE
[GR-1136] Adjustable cutoff lines

### PR DESCRIPTION
**Pull request by avarsava**

### CODE REVIEW

File: application/dash_application/views/SARS-CoV-2.py [hunk 1]
```diff
@@ -198,7 +200,7 @@ def generate_average_coverage_scatter(current_data, graph_params):
         graph_params["colour_by"],
         graph_params["shape_by"],
         graph_params["shownames_val"],
-        [(avg_coverage_cutoff_label, avg_coverage_cutoff_value)] # Should be unchanging
+        [(avg_coverage_cutoff_label, graph_params["avg_coverage_cutoff"])] # Should be unchanging
```
Comment (apmasell) [inline application/dash_application/views/SARS-CoV-2.py]: I think this comment and the one below is out of date.

**Review by apmasell — APPROVED** [review]

File: application/dash_application/views/SARS-CoV-2.py (hunk 1 above)
Comment (avarsava) [reply application/dash_application/views/SARS-CoV-2.py]: Fixed!

File: application/dash_application/views/SARS-CoV-2.py [hunk 2]
```diff
@@ -399,7 +407,7 @@ def layout(query_string):
                                 df,
                                 RAW_DATA_COLUMNS,
                                 [
-                                    (avg_coverage_cutoff_label, BEDTOOLS_CALC_COL.MeanCoverage, avg_coverage_cutoff_value,
+                                    (avg_coverage_cutoff_label, BEDTOOLS_CALC_COL.MeanCoverage, initial["avg_coverage_cutoff"],
```
Comment (slazicoicr) [inline application/dash_application/views/SARS-CoV-2.py:410]: The failure table is missing the failed on target.

Comment (avarsava) [reply application/dash_application/views/SARS-CoV-2.py:410]: Fixed

**Review by slazicoicr — APPROVED** [review]

**Pull request merged**